docs/api.md

```diff
@@ -0,0 +1,95 @@
+-   Grundsätzlich: REST-APIs!
+
+-   ID == PrimaryKey oder UUID. Etwas das stabil bleibt
+
+-   "ticketNumber" ist die Bon Nummer die man sonst ausgedruckt bekommen
+    würde. Also etwas für den Menschen einfach Lesbares
+
+APIs für Admin-Interface
+========================
+
+Auth:
+-----
+
+-   Bearer-Token als HTTP-Header?! =\> Erstmal ohne Authentifizierung
+    bauen, aber im Hinterkopf behalten
+
+Authentifizierung durchführen (kommt erst später)
+-------------------------------------------------
+
+HTTP-GET /places/\<place-id\>/bearerToken?password=\<pass\>
+
+> Answer: 200/OK\
+> JSON-Payload: { bearerToken: string; }
+
+ Warteschlangen anlegen
+----------------------
+
+> HTTP-POST: /places/\<place-id\>/queues\
+> Payload: JSON { queueName: string; }
+>
+> Answer: 200/OK;\
+> JSON-Payload: { id: string; queueName: string; }
+>
+> Answer: 4xx/Not ok
+>
+> Answer: 401/Not authorized
+
+Warteschlangen löschen
+----------------------
+
+> HTTP-DELETE: /places/\<place-id\>/queues/\<queue-id\>
+>
+> Answer: 200/OK
+>
+> Answer: 4xx/Not ok
+
+Zustand der Warteschlange abrufen
+---------------------------------
+
+> HTTP-GET: /places/\<place-id\>/queues?personDetails=full
+>
+> Answer: 200/OK;\
+> JSON-Payload: { id: string; name: string; entries: {id: string;
+> ticketNumber: number; name: string}\[\] }\[\]
```

Comment:
Wie ist das gemeint? entires ist ein dict mit id als key?
Wie erkenne ich dann die Reihenfolge? Anhand der ticketNumer?
Wäre ein Array, dass die Reihenfolge der Qeue vorgibt, nicht besser?

Reply:
entries ist ein array von dicts mit den keys 'id', 'ticketNumber' und 'name'

Reply:
Glaube entries: {id: string; ticketNumber: number; name: string} [ ] soll schon ein Array sein. In dem Array stecken dann Objekte vom Typ {id: string; ticketNumber: number; name: string}.